integration_tests/e2e/abilityToWork.cy.ts

```diff
@@ -19,7 +19,6 @@ context('Ability to work page', () => {
     cy.task('stubPlpPrisonListPageUi')
     cy.task('stubGetFrontEndComponents')
     cy.task('getPrisonerById')
-    cy.task('getCiagPlan')
```

Comment:
No need to stub retrieval of the CIAG Plan (Induction) when its a create journey cypress test